### DISCUSSION TITLE
UX - Copy To / Move  idea to another board

### DISCUSSION THREAD

**Discussion by x2zdigital** [Leantime/leantime#593] [Ideas]
Enable changing the location of an idea to another board
Copy an idea to another board

**Comment by marcelfolaron** [comment]
Duplicating the project is indeed possible already. You can see a button on
the top right hand corner on the project details screen

**Reply by x2zdigital** [reply]
Thanks, found it right after I posted, edited the post because i couldnt find the delete button :D 

I was expecting the function to be part of the main / sub navigation
